python/ray/data/tests/test_stats.py

```diff
@@ -454,50 +454,6 @@ def test_dataset_stats_basic(ray_start_regular_shared, enable_auto_log_stats):
             )
 
 
-def test_dataset_stats_stage_execution_time(ray_start_regular_shared):
-    # Disable stage/operator fusion in order to test the stats
-    # of two different map_batches operators without fusing them together,
-    # so that we can observe different execution times for each.
-    ctx = ray.data.DataContext.get_current()
-    curr_optimizer_enabled = ctx.optimizer_enabled
```

Comment:
remove this test because it overrides execution optimizer flag, but now ray data expects that optimizer is always enabled here: https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/execution/legacy_compat.py#L209